Projects/Contracts/Sources/ContractsStore.swift

```diff
@@ -0,0 +1,109 @@
+import Apollo
+import Flow
+import Presentation
+import hCore
+import hGraphQL
+
+public struct ContractState: StateProtocol {
+    public init() {}
+
+    var contractBundles: [ActiveContractBundle] = []
+    var contracts: [Contract] = []
+}
+
+public enum ContractAction: ActionProtocol {
+    // Fetch contracts for terminated
+    case fetchContractBundles
+    case fetchContracts
+
+    case setContractBundles(activeContractBundles: [ActiveContractBundle])
+    case setContracts(contracts: [Contract])
+    case goToMovingFlow
+    case goToFreeTextChat
+
+    #if compiler(<5.5)
+        public func encode(to encoder: Encoder) throws {
+            #warning("Waiting for automatic codable conformance from Swift 5.5, remove this when we have upgraded XCode")
+            fatalError()
+        }
+
+        public init(
+            from decoder: Decoder
+        ) throws {
+            #warning("Waiting for automatic codable conformance from Swift 5.5, remove this when we have upgraded XCode")
+            fatalError()
+        }
+    #endif
+}
+
+public final class ContractStore: StateStore<ContractState, ContractAction> {
+    @Inject var client: ApolloClient
+
+    public override func effects(
+        _ getState: () -> ContractState,
+        _ action: ContractAction
+    ) -> FiniteSignal<ContractAction>? {
+        switch action {
+        case .fetchContractBundles:
+            return
+                client.fetch(
+                    query: GraphQL.ActiveContractBundlesQuery(
+                        locale: Localization.Locale.currentLocale.asGraphQLLocale()
+                    ),
+                    cachePolicy: .fetchIgnoringCacheData
+                )
+                .map { data in
+                    data.activeContractBundles.map { ActiveContractBundle(bundle: $0) }
+                }
+                .map { activeContractBundles in
+                    ContractAction.setContractBundles(activeContractBundles: activeContractBundles)
+                }
+                .valueThenEndSignal
+        case .setContractBundles(let activeContractBundles):
```

Comment:
```suggestion
        default:
          break
```

use a default clause for non handled cases maybe?